pulsar-broker/src/test/java/org/apache/pulsar/client/api/InterceptorsTest.java

```diff
@@ -151,12 +151,12 @@ public void close() {
 
             @Override
             public Message<String> beforeSend(Producer<String> producer, Message<String> message) {
-                throw new NullPointerException();
+                throw new IllegalArgumentException();
```

Comment:
Maybe NotImplementException is better?

Reply:
This test is testing the interceptor throws exceptions, so it's ok for any runtime exceptions.